deploy/helm/kubecf/templates/db-seeder.yaml

```diff
@@ -0,0 +1,109 @@
+{{- if .Values.features.embedded_database.enabled -}}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: db-seeder
+spec:
+  template:
+    spec:
+      containers:
+      - name: seeder
+        image: {{ (index .Values "percona-xtradb-cluster").image.repository }}:{{ (index .Values "percona-xtradb-cluster").image.tag }}
+        env:
+        - name: DATABASE_ROOT_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Release.Name }}-pxc
+              key: mysql-root-password
+        - name: DATABASE_HOST
+          value: {{ .Release.Name }}-pxc
+        - name: NAMESPACE
+          value: {{ .Release.Namespace }}
+        volumeMounts:
+        - name: cloud-controller-database-password
+          mountPath: "/passwords/cloud_controller"
+          readOnly: true
+        - name: diego-database-password
+          mountPath: "/passwords/diego"
+          readOnly: true
+        - name: network-connectivity-database-password
+          mountPath: "/passwords/network_connectivity"
+          readOnly: true
+        - name: network-policy-database-password
+          mountPath: "/passwords/network_policy"
+          readOnly: true
+        - name: routing-api-database-password
+          mountPath: "/passwords/routing-api"
+          readOnly: true
+        - name: uaa-database-password
+          mountPath: "/passwords/uaa"
+          readOnly: true
+        - name: locket-database-password
+          mountPath: "/passwords/locket"
+          readOnly: true
+        - name: credhub-database-password
+          mountPath: "/passwords/credhub"
+          readOnly: true
+        command:
+        - "/bin/bash"
+        - "-ce"
+        - |
+          echo "Starting seeding..."
```

Comment:
Could you create a `.sh` file with these contents under https://github.com/SUSE/kubecf/tree/master/deploy/helm/kubecf/assets/scripts/jobs, please? This way we take advantage of syntax highlighting and linting.

Reply:
done